undertow/src/main/java/org/wildfly/extension/undertow/filters/FilterRefCapability.java

```diff
@@ -0,0 +1,50 @@
+package org.wildfly.extension.undertow.filters;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.function.Function;
+
+import org.jboss.as.clustering.controller.Capability;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.capability.RuntimeCapability;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+
+class FilterRefCapability implements Capability {
```

Comment:
What is the purpose of this class?  It does not seem to be referenced anywhere.

Reply:
ha. power out, thought it did not save and must have picked another name.